packages/authentication/src/drivers/crypto/hashes.js

```diff
@@ -0,0 +1,8 @@
+// External dependencies
+const { createHash } = require('crypto');
+// Internal dependencies
+const { crypto } = require('../../../config');
+
+const hashMD5 = createHash(crypto.hashAlgorithm);
+
+module.exports = hashMD5;
```

Comment:
You named your directory as the library crypto and as the config file.
You could change something like cryptoConfig. If it is clear for you, I don't have any problem